javascript/packages/utils/src/tableCli.ts

```diff
@@ -54,15 +55,37 @@ interface TableCreationProps {
   wordWrap?: boolean;
 }
 
+export type LogType = "json" | "text" | "table" | "silent";
```

Comment:
```suggestion
export type LogType =  "text" | "table" | "silent";
```
`json` isn't supported yet :)